File: 01-Assignment-02-02-2026/Using_C/program_9.c
/* Program 9: Find the average of 5 numbers using while loop */

#include <stdio.h>

int main() {
    float sum=0.0,nums[5] = {1,23,6.5,8,-10};
    int i = 0;
    while (i<5)
    {
        sum+=nums[i];
        i++;
    }
    printf("The Average of Nums is : %.2f",sum/5);
    
    
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program9.java
/*
Program 9
Java program to find the average of 5 numbers using while loop
*/

import java.util.Arrays;

public class Program9 {

    public static void main(String[] args) {
        double[] numbers = {1,6,3.6,8,9};

        System.out.println("The Numbers are " + Arrays.toString(numbers));

        int i = 0 ;
        double sum = 0;
        while (i < 5) {
            sum += numbers[i];
            i++;
        }
        System.out.println("Average of Above No. is " + ( sum / 5.0));
        
    }

}
